packages/browser/src/UIAddressInput.tsx

```diff
@@ -100,6 +101,16 @@ export function UIAddressInputInternal({
         resetInputHeight,
     } = useAutogrowTextView(textInputRef, onHeightChange, MAX_INPUT_NUM_OF_LINES);
 
+    const [isNoticeVisible, setNoticeVisible] = React.useState(false);
+
+    const onMaxLength = () => {
```

Comment:
Let's wrap it with `useCallback` or use it inline instead

Reply:
@savelichalex Yes, I just want to, just for now I'm trying to turn on ESLint... Something is wrong in my VSCode :)

Reply:
@savelichalex I will re-request a review when I finish this